### DISCUSSION TITLE
[Enhancement] select 组件在 filterable && disabled && multiple 状态时,会存在多余的换行

### DISCUSSION THREAD

**Discussion by supperdsj** [element-plus/element-plus#9559] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `Select`

## select 组件在 filterable && disabled && multiple 状态时,会存在多余的换行
select 组件在 filterable && disabled && multiple 状态时,在宽度不足的情况下会存在多余的换行,组件需要添加 :filterable="!disabled" 来解决,理论上在禁用组件的状态下 filterable 必不可用

**Comment by kooriookami** [comment]
已解决
